### PR TITLE
Add _raw suffix to non-normalized destination table names

### PR DESCRIPTION
**Pull request by sherifnada**
## What
* Adds the `_raw` suffix to all non-normalized table names in the destination 
* writes tables in the same schema/dataset as the table being normalized.

I'm pretty unhappy with this implementation and think we should remove it going forward, but putting it up to minimize delivery risk. Some better ways to do this are having DBT rename the raw tables in a pre-hook to add the _raw suffix, or having the destination integration output a "writtenCatalog" which contains the names of the tables it actually wrote. The latter is better in the case that we want to stop overwriting tables when performing full refresh syncs and instead write to a different table e.g `table_timestamp`.

### CODE REVIEW

File: airbyte-integrations/bases/base-normalization/dbt-project-template/dbt_project.yml [hunk 1]
```diff
@@ -35,6 +35,4 @@ quoting:
 # are materialized, and more!
 models:
   airbyte:
-    # Schema (dataset) defined in profiles.yml is concatenated with schema below for dbt's final output
```
Comment (cgardens) [inline airbyte-integrations/bases/base-normalization/dbt-project-template/dbt_project.yml:38]: eh?

**Review by cgardens — APPROVED** [review]
i know the goal is to get this in right now so feel free to skip this suggestion for now. but i think your life gets a whole lot easier if you just had a helper method that looks like this

```
  public String getRawTableName(String streamName) {
    return streamName + "_raw";
  }
```

it saves us needing put "_raw" all over the codebase. ideally that string should be written in exactly one place. (at least in java)

maybe also a function that maps (rawTableName) => streamname.

anyway. go for it.

File: airbyte-integrations/bases/base-normalization/dbt-project-template/dbt_project.yml (hunk 1 above)
Comment (sherifnada) [reply airbyte-integrations/bases/base-normalization/dbt-project-template/dbt_project.yml:38]: this suffixed the schema/dataset with `_NORMALIZED`

Comment (cgardens) [reply airbyte-integrations/bases/base-normalization/dbt-project-template/dbt_project.yml:38]: ah okay.

**Comment by sherifnada** [comment]
@cgardens good suggestion - implemented

**Pull request merged**